src/list-grid.scss

```diff
@@ -0,0 +1,232 @@
+@import './new-settings';
+@import './mixins';
+
+/*!
+.fd-grid-list
+*/
+$block: #{$fd-namespace}-grid-list;
+
+.#{$block} {
+  $fd-grid-list-border-radius: var(--sapElement_BorderCornerRadius);
+  $fd-grid-list-active-text-color: var(--sapList_Active_TextColor);
+  $fd-grid-list-item-box-shadow: var(--sapContent_Shadow0);
+  $fd-grid-list-mode-box-shadow: 0 0 0 0.125rem var(--sapList_SelectionBorderColor);
+  $fd-grid-list-highlight: (
+    'success': ('backgroundColor': var(--sapSuccessBorderColor)),
+    'error': ('backgroundColor': var(--sapErrorBorderColor)),
+    'warning': ('backgroundColor': var(--sapWarningBorderColor)),
+    'neutral': ('backgroundColor': var(--sapNeutralBorderColor))
+  );
+
+  @include fd-reset();
+
+  display: flex;
+  flex-direction: column;
+
+  .fd-list {
```

Comment:
I think you should consider getting rid of `fd-list` usage. The only things that are taken from list are interractive states (hover/active/select). I know that it's named grid-**list**. But now you overwrite almost every element, which will force users to put 2 classes for 1 element (`fd-list__x` and `fd-grid-list__x`), For me you can export interractive states to some mixin(just to keep same for list and grid list) and treat grid list as a standalone component. Let me know what you think about it, or for some more clarifications

Reply:
+1

Reply:
agree with you
but about to move interactive states to mixin I think it needs to be done by separate PR
1 PR = 1 component